targets/PythonSdk/make.js

```diff
@@ -1,10 +1,27 @@
 var path = require("path");
+var fs = require("fs");
 
 // Making resharper less noisy - These are defined in Generate.js
 if (typeof (templatizeTree) === "undefined") templatizeTree = function () { };
 if (typeof (getCompiledTemplate) === "undefined") getCompiledTemplate = function () { };
 
+function copyOverScripts(sourceDir, apiOutputDir)
+{
+    var srcDir = sourceDir + "\\source";
+    var outDir = apiOutputDir.substr(0, 18);
```

Comment:
Uh... whut?
There's no explanation here, and this seems unusual.
What are you doing here?
This will be very bizarre if your root-repo-path is different from mine (and it probably is)